### PR TITLE
Set interfaces to UP state

### PR DESCRIPTION
**Pull request by e0ne**
Let's set interfaces to UP state in Resource Manager to prevent
issues if CNIs don't do it.

Signed-off-by: Ivan Kolodyazhny <ikolodiazhny@nvidia.com>

### CODE REVIEW

File: pkg/resources/resources_manager.go [hunk 1]
```diff
@@ -178,6 +178,18 @@ func (rm *resourceManager) InitServers() error {
 		log.Printf("Resource: %+v\n", config)
 		devices := rm.GetDevices()
 		filteredDevices := rm.GetFilteredDevices(devices, &config.Selectors)
+		for _, device := range filteredDevices {
```
Comment (adrianchiris) [inline pkg/resources/resources_manager.go:181]: Should this be the responsibility of device plugin ?

Comment (e0ne) [reply pkg/resources/resources_manager.go:181]: It's only one common place of all device plugin consumers. To implement it in CNI we need to implement it in all CNIs. The same for operator: we can't fix all operators(we've got only Network Operator now but it doesn't contain daemon the each node)

Comment (moshe010) [reply pkg/resources/resources_manager.go:181]: This is a workaround just to fix vmware issue. As Ivan mention solving it in the operator level now it a bit problematic.
@e0ne  maybe you can add a comment in the code on this?

Comment (e0ne) [reply pkg/resources/resources_manager.go:183]: @moshe010 added a comment

**Pull request merged**